packages/dialtone-vue2/components/rich_text_editor/extensions/mentions/MentionComponent.vue

```diff
@@ -5,7 +5,7 @@
   >
     <dt-link
       kind="inverted"
-      class="dt-link--mention"
+      class="d-link--mention"
```

Comment:
Will need to be `class="d-link d-link--mention"`, unless we'd prefer to allow `class="d-link--mention` available standalone (slight deviation we sometimes do from "true" BEM), which means you'd have to ensure the base `d-link` styles fold into `--mention` too.

Reply:
ah yep thanks, definitely should have d-link

Reply:
And duh, that's what we do for other variants. 
<img width="451" alt="image" src="https://github.com/dialpad/dialtone/assets/1165933/c35d38d5-2eaa-4c7a-b7ee-6c4a00027840">